corehq/apps/app_manager/templates/app_manager/partials/settings/multimedia_ajax.html

```diff
@@ -48,6 +49,46 @@ <h4 class="panel-title panel-title-nolink">{% trans 'Download Multimedia Zip' %}
       </form>
     </div>
   {% endif %}
+
+  {% if request|toggle_enabled:"MULTI_MASTER_LINKED_DOMAINS" %}
+    <div class="panel panel-appmanager">
+      <div class="panel-heading">
+        <h4 class="panel-title panel-title-nolink">{% trans 'Copy Multimedia from Another Application' %}</h4>
+      </div>
+      <div class="panel-body">
+        {% if import_apps %}
+          <form action="{% url "copy_multimedia" domain app.id %}" method="POST">
+            <p class="help-block">
+              Select an app to copy over any media that is present in the selected app but missing in this app.
+            </p>
+            <p>
+              {% csrf_token %}
+              <select name="app_id" class="form-control hqwebapp-select2"
+                      data-placeholder="{% trans_html_attr "Select an app" %}">
+                <option value=""></option>
+                {% for app in import_apps %}
+                  {# Poor readability because any whitespace within the option will also appear in the title attribute and therefore on hover #}
+                  <option value="{{ app.id }}">{{ app.name }} ({% for id, count in import_app_counts.items %}{% if id == app.id %}{% blocktrans with c=count %}{{ c }} items to import{% endblocktrans %}{% endif %}{% endfor %})</option>
```

Comment:
might want to make this "item(s) to import" to adjust in case of 1 item as in example

Reply:
d4f5c738a7d19aa4b3917a895ee28f00c766bd21